README.md

```diff
@@ -1,39 +1,78 @@
 
-* **TeamCity:** https://teamcity.gutools.co.uk/viewType.html?buildTypeId=Subscriptions_ContentAuthorisationProxy
-* **RiffRaff:** https://riffraff.gutools.co.uk/deployment/history?projectName=content-authorisation-proxy
+Find out if a download of the digital edition is allowed. 
 
-## General Setup
+1) When does this user's trial expire?
+Check with the Dynamo table that keeps track by calling `/auth`
+
+Look in `DEV.public.conf` to see which table we are referring to when running locally.
+
+```
+POST /auth HTTP/1.1
+Host: localhost:9300
+Content-Type: application/json
+Cache-Control: no-cache
 
-1. Setup AWS credentials (we use the gu-membership account)
 
-   Ask your teammate to create an account for you and securely send you the access key. For security, you must enable [MFA](http://aws.amazon.com/iam/details/mfa/).
+{
+    "appId": "<appId>",
+    "deviceId": "<deviceId>"
+}
+```
+Sample response:
+```
+{
+  "expiry": {
+    "expiryType": "free",
+    "provider": "default",
+    "expiryDate": "2017-05-04"
+  }
+}
+```
 
-   In `~/.aws/credentials` add the following:
+2) Can the user download the digital edition because they are subscribed?
+`/subs` will check with Zuora to see when the subscription expires. 
+```
+POST /subs HTTP/1.1
+Host: localhost:9300
+Content-Type: application/json
+Cache-Control: no-cache
 
-   ```
-   [membership]
-   aws_access_key_id=[YOUR_AWS_ACCESS_KEY]
-   aws_secret_access_key=[YOUR_AWS_SECRET_ACCESS_KEY]
-   ```
+{
+    "appId": <appId>,
+    "deviceId": <deviceId>,
+    "subscriberId": <Id from Zuora. From Zuora dev if running locally>,
+    "password": <password>
+}
 
-   In `~/.aws/config` add the following:
+```
+Sample response:
+```
+{
+  "expiry": {
+    "expiryType": "sub",
+    "expiryDate": "2018-04-19",
+    "content": "SevenDay"
+  }
+}
+```
 
-   ```
-   [default]
-   output = json
-   region = eu-west-1
-   ```
+## Running the app locally
 
-1. Download our private keys from the `subscriptions-private` S3 bucket. You will need an AWS account so ask another dev.
+1. Setup AWS credentials (we use the gu-membership account)
+
+   You need membership AWS credentials from Janus. 
+   
+2. Download our private keys from the `subscriptions-private` S3 bucket. 
 
     If you have the AWS CLI set up you can run
     ```
-    aws s3 cp s3://subscriptions-private/DEV/cas-proxy.conf /etc/gu --profile membership
-
+    aws s3 cp s3://subscriptions-private/DEV/cas-proxy.private.conf /etc/gu --profile membership
     ```
+3. Run `sbt` then `devrun`.  
     
-## Automated tests
+## Links
 
-https://www.runscope.com/radar/f862w29p8z5f/7ebb04c6-4b2d-4c25-a4d8-11d641faf2d2/overview
+* **Automated tests on Runscope:** https://www.runscope.com/radar/f862w29p8z5f/7ebb04c6-4b2d-4c25-a4d8-11d641faf2d2/overview
+* **TeamCity:** https://teamcity.gutools.co.uk/viewType.html?buildTypeId=Subscriptions_ContentAuthorisationProxy
```

Comment:
This teamcity url is out of date as well, the new one is
https://teamcity-aws.gutools.co.uk/viewType.html?buildTypeId=memsub_subscriptions_ContentAuthorisationProxy